scripts/ci/travis.sh

```diff
@@ -1,12 +1,49 @@
 #!/usr/bin/env bash
 
-set -euvo pipefail
+set -euo pipefail
 
+# Ensure DC is available
+export DC="$(command -v ${DC:-})"
+command -v "$DC" >/dev/null || \
+  { echo >&2 '[ERROR] "$DC" should be available'; exit 1; }
+
+# Ensure that the globally installed 'dub' is not implicitly available for the testsuite.
+#
+# 1/3 Rename it to "$HOST_DUB" and hide it:
+export HOST_DUB
+HOST_DUB="$(command -v dub)" ||
+  { echo >&2 "[ERROR] 'dub' should be available"; exit 1; }
+unset DUB
+hash -p /dev/null/dub dub
+
+# 2/3 Verify that both 'dub' and "$DUB" are not available:
+dub > /dev/null 2>&1 || command -v "${DUB:-}" >/dev/null && \
+  { echo >&2 "[ERROR] 'dub' shouldn't be available"; exit 1; }
```

Comment:
shouldn't be available "anymore" ?
Otherwise the error message might be confusing.